src/systems/userinput/bindings/keyboard-mouse-user.js

```diff
@@ -908,7 +915,17 @@ export const keyboardMouseUserBindings = addSetsToBindings({
       priority: 101
     },
     {
-      src: [stopInspectingViaMouse, stopInspectingViaKeyboardEscape, stopInspectingViaKeyboardFocusRelease],
+      src: { value: paths.device.keyboard.key("i") },
+      dest: { value: stopInspectingSelf },
+      xform: xforms.rising
```

Comment:
For consistency, it might be better to use `falling` but personally I don't like the idea of having to hold the `i` key while orbiting.